app/results/page.jsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import Products from './products';
+import { useSearchParams } from 'next/navigation';
+import { useState, useEffect } from 'react';
+
+export default function Page() {
+  const searchParams = useSearchParams();
+  const [query, setQuery] = useState('');
+  // Sets current page to inital searchParams, default value is 1
+  const [currentPage, setCurrentPage] = useState(
+    Number(searchParams.get('page'))
+  );
+
+  // UseEffect to handle inital state of query based on searchParams
+  useEffect(() => {
+    // If query is exists in searchParams, update query state accordingly
+    if (searchParams.get('query')) {
```

Comment:
More for my own benefit than anything. At line 18, you check if searchParams.get('query') is truthy. Do you need the ? operator for the .toString() method if it is already truthy?

Reply:
I don't think it hurts anything, just wondering if there is an extra reason I'm unaware of for a cool javascript oddity. :)

Reply:
I think I ended up just leaving it there before I added the truthy statement. It's fine to remove it if you wish.